src/System.Drawing.Primitives/tests/SizeFTests.netcoreapp.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using Xunit;
+
+namespace System.Drawing.PrimitivesTest
+{
+    public partial class SizeFTests
+    {
+        #region SizeF * float tests
+
+        private void MultSizeFFloatTester(float width, float height, float multiplier)
+        {
+            SizeF sz = new SizeF(width, height);
+            SizeF mulExpected;
+
+            mulExpected = new SizeF(width * multiplier, height * multiplier);
+
+            Assert.Equal(mulExpected, sz * multiplier);
+            Assert.Equal(mulExpected, multiplier * sz);
+        }
+
+        [Theory]
+        [InlineData(1000.234f, 2400.933f)]
+        [InlineData(float.MaxValue, 0.0f)]
+        [InlineData(float.MaxValue, 1.0f)]
+        [InlineData(float.MaxValue, -1.0f)]
+        [InlineData(float.MaxValue, float.MaxValue)]
+        [InlineData(float.MinValue, 0.0f)]
+        [InlineData(float.MinValue, 1.0f)]
+        [InlineData(float.MinValue, -1.0f)]
+        [InlineData(float.MinValue, float.MinValue)]
+        [InlineData(float.MaxValue, float.MinValue)]
+        [InlineData(float.MinValue, float.MaxValue)]
+        public void MultiplicationTest(float value1, float value2)
+        {
+            MultSizeFFloatTester(value1, 1.0f, value2);
```

Comment:
> a test case which covers (3.0f, 4.0f) * 5.0f, for example.
There is already a variation of that sort this line 26 as mentioned below.
```C#
       [InlineData(1000.234f, 2400.933f)]
```
Just that this works by instantiating two size objects (1000.234f, 1.0f) and (1.0f, 1000.234f) both being multiplied by 2400.933f. Having both varying at the same time is just another variation, but really not necessary when the individual components of Size/F object have been sufficiently covered.

As for 'readability', I think I would like to differ on that as the test cases are quite readable. There are far more complex test cases in this very repository wherein the test cases are based on intricate framework. One such being at [this link](https://github.com/dotnet/corefx/pull/18109/files) on which I worked!

So are we clear for accepting the changes or we want to seek opinion from wider audience?

Thanks Mandar